prime-router/src/main/kotlin/report/ReportService.kt

```diff
@@ -22,6 +26,13 @@ class ReportService(
             ?: reportGraph.db.fetchReportFile(childReportId)
     }
 
+    fun getRootItemIndex(childReportId: UUID, childIndex: Int): Int? {
```

Comment:
Won't comment again to reduce verbosity but we should consider kdocs for a lot of these if nothing else to keep with good practice.